### ISSUE TITLE
[BUG]cron expression save as wrong value

### ISSUE THREAD

**Issue by FrankLeeC** [cweijan/vscode-mysql#490]
- OS: Windows11
- Database Type: MySQL

scripts:
```sql
create table text_cron (
    cron text not null
);

insert into text_cron (cron) values ('*/1 * * * *');

select cron, LENGTH(cron) from text_cron;
```

result:
|cron|LENGTH(cron)|
|-|-|
|1 * * * *|9|

LENGTH(cron) should be 11, so it's not a visual problem and the data was really changed before inserting into database.

**Comment by cweijan** [comment]
This may be related to encoding, have you tried doing this with other tools?

**Comment by FrankLeeC** [comment]
> This may be related to encoding, have you tried doing this with other tools?

Unfortunately, I just tried in mysql terminal with the same test case, and it works correctly in terminal.

**Comment by cweijan** [comment]
Thanks for your feedback, I will try to fix this.

**Comment by cweijan** [comment]
Fix at version 4.7.4, the reason for this error is because */ is incorrectly parsed as comment.
